content/millennium/r4/scheduling/appointment.md

```diff
@@ -312,7 +318,39 @@ _Implementation Notes_
 
 <%= definition_table(:appointment, :create, :r4) %>
 
-### Example
+### Example - Proposed Appointment
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment
+
+#### Body
+
+  <%= json(:r4_appointment_request_create) %>
```

Comment:
let's change this to r4_proposed_appointment_create?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-1d7df8ceb9f0ae8471323516e037c662R329
https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-1a83ec9f4ba450cba6a16d8bcda1890cR366